spec/models/item_spec.rb

```diff
@@ -5,4 +5,14 @@
     expect(create(:item)).to be_valid
   end
 
+  it 'has a status of OOS if condition is OOS' do
+    item = build :item, condition: 'Out of Service', status: 'Unassigned'
+    expect(item.status).to eq('Out of Service')
+  end
+
+  it 'has a status of Unassigned if condition is Available and status is Unassigned' do
+    item = build :item, condition: 'Available', status: 'Unassigned'
+    expect(item.status).to eq('Unassigned')
```

Comment:
Agreed, something went awry. Will try to figure out where I was going with this.